app/server.js

```diff
@@ -1,12 +1,12 @@
 "use strict";
 
-const http = require("http"),
+const https = require("https"),
   express = require("express");
 
 const ServerFactory = function(config, logger) {
 
   const app = express(),
-    server =  http.createServer(app);
```

Comment:
@rodrigopavezi I'm fine with just https, just wanted to discuss the potential of listening for both.